devtools/src/run-init-debug.js

```diff
@@ -7,3 +9,9 @@ if (initDebug().preExistingArcs) {
     }]
   }));
 }
+
+initPlannerComponent();
+
+window.postMessage('debug-inited', '*');
```

Comment:
I had to add this because I can't send messages to the page until the page is 'booted'. Is there a cleaner way?